blendergltf.py

```diff
@@ -250,7 +250,12 @@ def export_buffer(self, settings):
         if settings['buffers_embed_data']:
             uri = 'data:text/plain;base64,' + base64.b64encode(data).decode('ascii')
         else:
-            uri = bpy.path.clean_name(self.name) + '.bin'
+            if settings['buffers_combine_data']:
+                parts = os.path.splitext(os.path.basename(settings['filepath']))
```

Comment:
This would break for anything using blendergltf as a module instead of as an addon since 'filepath' is not in the default settings.  In this case it happened to get copied into the setting by the addon by mistake. I am not opposed to adding a 'gltf_output_name' setting that can be used as a basename. I think a reasonable default would be 'data' or 'gltf'. Although it might be a little weird to add a setting that currently only would be used for naming combined buffers. The addon can handle this setting without exposing it to the UI to keep things simpler for the user.